db/migrate/20160902111740_initial_schema.rb

```diff
@@ -49,5 +51,29 @@
       column :public_access, TrueClass
       column :content_type, :repository_content_type
     end
+
+    create_table :commits do
+      primary_key :id
+      foreign_key :repository_id, :repositories, index: true, null: false
+      foreign_key :author_id, :users, index: true, null: true
+      foreign_key :editor_id, :users, index: true, null: true
+
+      column :author_name, String
+      column :editor_name, String
```

Comment:
Oh and it would be nice to have a `pusher`, i.e. a User that has pushed this commit. This can be a simple many_to_one association to the User model (not optional).